js/ui/components/clubhouse.js

```diff
@@ -67,10 +67,6 @@ var ClubhouseNotification = new Lang.Class({
     Extends: NotificationDaemon.GtkNotificationDaemonNotification,
 
     _init: function(source, notification) {
-        this._currentIconPath = null;
-
-        notification.title = GLib.Variant.new('s', '');
```

Comment:
This is actually used. Because in the Clubhouse I was not setting any title (and without a title, it won't work).
Maybe we can change it so that we just set the empty title if none is coming in the notification...

Reply:
The simplest thing to do is to ensure the clubhouse conforms to the expectations the shell has, so I added a patch to set an empty title there. I don't think we need to change the shell expectations in this case.